tests/integration-tests/tests/update/test_update.py

```diff
@@ -1085,59 +1109,36 @@ def test_dynamic_file_systems_update(
     )
 
 
+@pytest.mark.usefixtures("external_shared_storage_stack")
 def _create_shared_storages_resources(
     snapshots_factory,
     request,
     vpc_stack,
     region,
-    efs_stack_factory,
-    efs_mount_target_stack_factory,
-    fsx_factory,
-    svm_factory,
-    open_zfs_volume_factory,
     bucket_name,
 ):
+
     """Create existing EBS, EFS, FSX resources for test."""
     # create 1 existing ebs
-    ebs_volume_id = snapshots_factory.create_existing_volume(request, vpc_stack.cfn_outputs["PublicSubnetId"], region)
-
-    # create 1 efs
-    existing_efs_ids = efs_stack_factory(1)
-    efs_mount_target_stack_factory(existing_efs_ids)
-    existing_efs_id = existing_efs_ids[0]
+    Public = vpc_stack.cfn_outputs["PublicSubnetId"]
+    ebs_volume_id = snapshots_factory.create_existing_volume(request, Public, region)
 
-    # create 1 fsx lustre
+    # external-shared-storage-stack params
+    vpc = vpc_stack.cfn_outputs["VpcId"]
+    PublicAz3 = vpc_stack.cfn_outputs["PublicAz3SubnetId"]
+    PublicAz2 = vpc_stack.cfn_outputs["PublicAz2SubnetId"]
     import_path = "s3://{0}".format(bucket_name)
     export_path = "s3://{0}/export_dir".format(bucket_name)
-    existing_fsx_lustre_fs_id = fsx_factory(
-        ports=[988],
-        ip_protocols=["tcp"],
-        num=1,
-        file_system_type="LUSTRE",
-        StorageCapacity=1200,
-        LustreConfiguration=LustreConfiguration(
-            title="lustreConfiguration",
-            ImportPath=import_path,
-            ExportPath=export_path,
-            DeploymentType="PERSISTENT_1",
-            PerUnitStorageThroughput=200,
-        ),
-    )[0]
-
-    # create 1 fsx ontap
-    fsx_ontap_fs_id = create_fsx_ontap(fsx_factory, num=1)[0]
-    fsx_ontap_volume_id = svm_factory(fsx_ontap_fs_id, num_volumes=1)[0]
-
-    # create 1 open zfs
-    fsx_open_zfs_root_volume_id = create_fsx_open_zfs(fsx_factory, num=1)[0]
-    fsx_open_zfs_volume_id = open_zfs_volume_factory(fsx_open_zfs_root_volume_id, num_volumes=1)[0]
+
+    # create external-shared-storage-stack
+    storage_stack = external_shared_storage_stack(vpc=vpc, PublicAz3=PublicAz3, PublicAz2=PublicAz2, import_path=import_path, export_path=export_path, Public=Public)
 
     return (
         ebs_volume_id,
-        existing_efs_id,
-        existing_fsx_lustre_fs_id,
-        fsx_ontap_volume_id,
-        fsx_open_zfs_volume_id,
+        storage_stack.cfn_outputs["efs_id"],
+        storage_stack.cfn_outputs["fsx_lustre_fs_id"],
+        storage_stack.cfn_outputs["fsx_ontap_volume_id"],
+        storage_stack.cfn_outputs["fsx_open_zfs_volume_id"]
```

Comment:
This line may introduce lint errors due to missing comma. Have a look at GH checks for the PR

Reply:
I am addressing formatting and style and will push that in my next commit